app/data/repositories/CardsRepository.py

```diff
@@ -0,0 +1,70 @@
+from app.data import databaseConnection
+from app.data.models.Card import Card
+
+
+conn = databaseConnection.connect()
+
+
+def createCard(deck_id, word, translation):
+    return Card.new_card(deck_id, word, translation)
```

Comment:
Card.create
and arent there constructor in python to just call Card(deck_id, word, translation)?

Reply:
There is but like I mentioned above I created those methods to be able to pass two different types of arguments. When I want to create an object of the card passing data from database using an array of arguments and when I create a new card when I don't have card.id because it's not yet in the database.